rabbitmq/windows/3.9.20/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM mcr.microsoft.com/windows/servercore:ltsc2019
```

Comment:
please change in ltsc2022

Reply:
Unfortunately that will lockdown the image to only host on ltsc2022 as minimum.
ltsc2019 will ensure it can run on both.

Still lots of companies are using 2019 hosts including AKS.

Shouldn't we keep it as is?